api/taxonomies/views.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import generics
+
+from api.taxonomies.serializers import TaxonomySerializer
+from api.base.views import JSONAPIBaseView
+
+import json
```

Comment:
Convention for import structure on the OSF (and I think more generally):

```
standard libraries

3rd party libraries

our modules
```
So in this case (and if you're chris seto you care about 📐 imports):
```
import json

from rest_framework import generics

from api.base.views import JSONAPIBaseView
from api.taxonomies.serializers import TaxonomySerializer
```